build.sbt

```diff
@@ -93,6 +93,53 @@ ThisBuild / headerLicense  := Some(HeaderLicense.Custom(
      |""".stripMargin
 ))
 
+ThisBuild / develocityConfiguration := {
+  def pubDev = Option(System.getProperty("scala.build.publishDevelocity"))
+  val isInsideCI = sys.env.get("JENKINS_URL").exists(_.contains("scala-ci.typesafe.com"))
+  val previousConfig = develocityConfiguration.value
+  val previousBuildScan = previousConfig.buildScan
+  val previousBuildCache = previousConfig.buildCache
+  val isBuildCacheEnabled = sys.BooleanProp.keyExists("build-cache.enabled") // same prop in Scala 3
```

Comment:
We don't have it in Scala 3, and I don't know if I am ever going to set it up.

Reply:
Ah, I looked at a different commit. Will align with Scala 3.